tests/scripts/thread-cert/node.py

```diff
@@ -469,7 +475,9 @@ def read_border_routing_counters_delta(self):
                     assert (key in new_counters)
                     value = [new_counters[key][0] - old_counters[key][0], new_counters[key][1] - old_counters[key][1]]
                     delta_counters[key] = value
-        delta_counters = {key: value for key, value in delta_counters.items() if value[0] and value[1]}
+        delta_counters = {
+            key: value for key, value in delta_counters.items() if not isinstance(value, int) and value[0] and value[1]
```

Comment:
Without `isinstance`, it should raise an error when old counters don't exist. Why didn't the old implementation cause an error?

Reply:
In the old implementation, all the values were `(int, int)` so it wouldn't cause an error.

Reply:
Makes sense. But this makes `read_border_routing_counters_delta` return only `inbound_unicast`, `inbound_multicast`, `outbound_unicast`, `outbound_multicast`. Is this expected?

Reply:
This is expected. For now we don't want to check the delta of RA/RS in a specific period.